conf/robottelo.yaml.template

```diff
@@ -60,3 +60,7 @@ ROBOTTELO:
   CDN: true
   # Run one datapoint or multiple datapoints for tests
   RUN_ONE_DATAPOINT: false
+  # Satellite version supported by this branch
+  # UNDR version is used for some URL composition
+  SATELLITE_VERSION: "6.9"
```

Comment:
Why we are hard-coding it to 6.9, we have utils to fetch the satellite version from satellite and being used everywhere in tests. What if user provides a wrong satellite version here in robottelo ? Do we have any plans to use this setting in robottelo ? 

Thoughts ?

Reply:
Because the master branch of robottelo only supports a single satellite y-stream version, at the moment that's 6.9.  This would get updated to 6.10 when the 6.9.z branch is created.

That being said, I recognize your comments about setting it from env, and we can do that in CI. 

The utils to fetch satellite version from test aren't of any use here, this is used to format dynaconf vars when they're read.  We also need to be able to collect tests without having a satellite reachable.

This is part of a larger plan to deduce reliance on `get_satellite_version` in the pytest session when you haven't necessarily checked out a satellite or don't have one available yet.

@JacobCallahan PR #8303 is establishing an xdist pattern for instance checkout on demand. It will be using this setting (he's actually looking at creating a new VERSION section for this), in order to default broker args for the checkout.